src/components/PopoverMenu.tsx

```diff
@@ -283,7 +283,7 @@ function PopoverMenu({
                 <View style={[isSmallScreenWidth ? {maxHeight: windowHeight - 250} : styles.createMenuContainer, containerStyles]}>
                     {renderHeaderText()}
                     {enteredSubMenuIndexes.length > 0 && renderBackButtonItem()}
-                    <WrapComponent contentContainerStyle={scrollContainerStyle}>
+                    <WrapComponent contentContainerStyle={shouldUseScrollView && scrollContainerStyle}>
```

Comment:
This is still causing errors in FAB

<img width="570" alt="Screenshot 2024-09-23 at 7 31 50 PM" src="https://github.com/user-attachments/assets/587bb945-ad35-4b76-84a2-566a30886f93">

Reply:
I updated.